semantic-conventions/src/opentelemetry/semconv/model/utils.py

```diff
@@ -33,7 +33,7 @@ def validate_id(semconv_id, position):
 def validate_values(yaml, keys, mandatory=()):
     """ This method checks only valid keywords and value types are used
     """
-    unwanted = list(set(yaml) - set(keys))
+    unwanted = [k for k in yaml.keys() if k not in keys]
```

Comment:
Note: this is a slight change to existing logic, but it is only used for formatting an exception, and the change I made is only to ensure a consistently-ordered message, so it can be safely asserted.